charts/celeborn/README.md

```diff
@@ -40,17 +43,57 @@ When you want to test the template rendering, but not actually install anything.
 There are two ways to render templates. It will return the rendered template to you so you can see the output.
 
 - Local rendering chart templates
+ 
 ```shell
 helm template --debug ../celeborn
 ```
+
 - Server side rendering chart templates
+
 ```shell
 helm install --dry-run --debug --generate-name ../celeborn
 ```
-More details in [Helm Install](https://helm.sh/docs/helm/helm_install/)
-The chart can be customized using the following [celeborn configurations](https://celeborn.apache.org/docs/latest/configuration/#important-configurations)
-Specify parameters using `--set key=value[,key=value]` argument to `helm install`
+
+More details in [Helm Install](https://helm.sh/docs/helm/helm_install/).
+The chart can be customized using the following [celeborn configurations](https://celeborn.apache.org/docs/latest/configuration/#important-configurations).
+Specify parameters using `--set key=value[,key=value]` argument to `helm install`.
 
 ## Documentation
 
-For additional details on deploying the Celeborn Kubernetes Helm chart, please refer to the [Celeborn on Kubernetes](https://celeborn.apache.org/docs/latest/deploy_on_k8s/) documentation
+For additional details on deploying the Celeborn Kubernetes Helm chart, please refer to the [Celeborn on Kubernetes](https://celeborn.apache.org/docs/latest/deploy_on_k8s/) documentation.
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` | Pod affinity |
+| cluster.name | string | `"cluster"` | Specifies Kubernetes cluster name |
+| dnsPolicy | string | `"ClusterFirst"` | Specifies the DNS policy for Celeborn pods to use |
+| fullnameOverride | string | `""` | String to override the default generated fullname |
+| hostNetwork | bool | `false` | Specifies whether to use the host's network namespace |
+| image.pullPolicy | string | `"Always"` | Image pull policy |
+| image.pullSecrets | list | `[]` | Image pull secrets for private image registry |
+| image.repository | string | `"aliyunemr/remote-shuffle-service"` | Image repository |
+| image.tag | string | `"0.1.1-6badd20"` | Image tag |
+| nameOverride | string | `""` | String to override the default generated name |
+| nodeSelector | object | `{}` | Pod node selector |
+| podAnnotations | object | `{}` | Pod annotations |
+| podMonitor.enable | bool | `true` | Specifies whether a PodMonitor should be created |
+| podMonitor.podMetricsEndpoint | object | `{"interval":"5s","portName":"metrics","scheme":"http"}` | Specifies PodMonitor endpoint |
+| priorityClass.master.create | bool | `false` | Specifies whether a new priority class for Celeborn master pods should be created |
+| priorityClass.master.name | string | `""` | Specifies the name of priority class for Celeborn master pods to be used (created if `create: true`) |
+| priorityClass.master.value | int | `1000000000` | Specifies the integer value of this priority class, default is half of system-cluster-critical |
```

Comment:
Could the description mention which values the priority class has?

Reply:
> Could the description mention which values the priority class has?

Added `empty means "${Release.Name}-master-priority-class"` in description.